.scivision/model.yml

```diff
@@ -1,5 +1,5 @@
 name: VEdge Detector
-url: https://github.com/MartinSJRogers/VEdge_Detector_scivision.git
+url: https://github.com/alan-turing-institute/VEdge_Detector_scivision/
 import: vedge_detector
```

Comment:
I think this should be https://github.com/MartinSJRogers/VEdge_Detector_scivision/